packages/compiler/src/server/server.ts

```diff
@@ -151,6 +155,51 @@ function main() {
   connection.listen();
 }
 
+function initializeConsoleLogger(logger: ConsoleLogger): void {
```

Comment:
this is more complex than I expected. isn't the console output sent to the lsp client by default?